### DISCUSSION TITLE
Cascade Filter and Hide-Body

### DISCUSSION THREAD

**Discussion by kookma** [TiddlyWiki/TiddlyWiki5#6360] [Q&A]
If I have a temp tiddler like `$:/temp/ptocedure` and then the cascades uses the $:/core/ui/ViewTemplate/body/code to display it based on $:/config/ViewTemplateBodyFilters/system

Now add the `hide-body`  field with `yes` value and you expect to have the body hides!
BUT nothing happened!

That means if I have a custom viewtemplate through cascades only one of them will work!

**Comment by pmario** [comment]
At the moment, the "hide-body" filter is number 4 in the cascade and "body/code" is number 1.. The first filter, that returns a value wins. ... 

I think it would be worth a discussion, if the "hide-body" should be number 1 by default ... @Jermolene ??

>That means if I have a custom viewtemplate through cascades only one of them will work!

You can create a cascade filter that selects a template, that also contains a cascade. You can add your logic, how your templates are activated. .... But in the end only 1 body template can be active depending on some filter logic.

**Comment by kookma** [comment]
Thanks Mario! As you suggested I added the logic for hide-body into my own filter!

> I think it would be worth a discussion, if the "hide-body" should be number 1 by default ... @Jermolene ??
It makes sense!
